ui/src/components/applications/Applications.vue

```diff
@@ -87,19 +87,19 @@
                   ? $t('remove_shortcut') : $t('add_shortcut')}}
                 </a>
               </li>
-              <li>
+              <li v-if="props.row.editable == 1 && !props.row.external && view.isRoot">
                 <a @click="props.row.pin == 1 ? removePin(props.row.id) : addPin(props.row.id)">
                   <span :class="['fa', 'fa-map-pin', 'action-icon-menu']"></span>
                   {{props.row.pin == 1
                   ? $t('remove_pin') : $t('add_pin')}}
                 </a>
               </li>
               <li
-                v-if="props.row.editable == 1 && !props.row.external && hideUninstall === 'disabled'"
+                v-if="props.row.editable == 1 && !props.row.external && view.isRoot"
```

Comment:
Here the condition `hideUninstall === 'disabled'` is necessary because if the prop is `enabled` we will see the divider whitout a "Remove" action below

Reply:
done